### PR TITLE
[BugFix][Build] ubuntu aarch64 use conda aarch64.sh #5053

### PR DESCRIPTION
**Pull request by songenjie**
#5053

### CODE REVIEW

**Comment by songenjie** [comment]
I just registered a personal account on Code Facebook

File: scripts/setup-ubuntu.sh [hunk 1]
```diff
@@ -109,10 +109,24 @@ function install_fbthrift {
 }
 
 function install_conda {
-  mkdir -p conda && cd conda
-  wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-  MINICONDA_PATH=/opt/miniconda-for-velox
-  bash Miniconda3-latest-Linux-x86_64.sh -b -p $MINICONDA_PATH
+  MINICONDA_PATH=/opt/miniconda-for-velox-jasong
+  if [ -d "${MINICONDA_PATH}" ]; then
+    if prompt "${MINICONDA_PATH} already exists. Delete?"; then
```
Comment (kgpai) [inline scripts/setup-ubuntu.sh:114]: Please leave out prompt from scripts that are used for automation. Also please remove personal paths.

**Review by kgpai — COMMENTED** [review]
Thanks !

**Comment by songenjie** [comment]
PTAL

**Pull request closed without merging**